src/components/DayPickerRangeController.jsx

```diff
@@ -249,7 +629,7 @@ export default class DayPickerRangeController extends React.Component {
 
   isDayAfterHoveredStartDate(day) {
     const { startDate, endDate, minimumNights } = this.props;
-    const { hoverDate } = this.state;
+    const { hoverDate } = this.state || {};
```

Comment:
When this function is called from the constructor, this.state has not yet been set. 

... this same response applies on the other areas too. 

It's a weird edge case, but this seemed like the cleanest solution?